packages/commonwealth/client/scripts/controllers/server/search.ts

```diff
@@ -243,19 +271,14 @@ class SearchController {
     }
   };
 
-  public searchMentionableAddresses = async (
+  public searchMentionableProfiles = async (
     searchTerm: string,
-    params: SearchParams,
-    order?: string[]
+    chainScope: string
   ) => {
-    const { pageSize, communityScope, chainScope } = params;
     try {
-      const response = await $.get(`${app.serverUrl()}/bulkAddresses`, {
```

Comment:
Does this deprecate bulkAddresses?

Reply:
Yes, `bulkAddresses` was used for quill mentions and search. Both have been updated to use `searchProfiles` instead.

Reply:
Feel free to remove the corresponding code!!

Reply:
Removed the dead code.